spec/models/incident_spec.rb

```diff
@@ -9,13 +9,13 @@
       it 'sends an email' do
         incident.supervisor.update email: 'supervisor@example.com'
 
-        expect { incident.notify_supervisor_of_new_report }.to(have_enqueued_job.on_queue('mailers'))
```

Comment:
They missed this one in the initializer.

`config.action_mailer.deliver_later_queue_name` now defaults to `nil` like all the other queues. The thinking is: the default should be just use one queue for everything. People won't mess up that default, and they can change it to`'mailers'` or whatever if their infrastructure _needs_ seperate queues.